components/backup-stream/src/endpoint.rs

```diff
@@ -238,7 +238,7 @@ where
         let safepoint_name = self.pause_guard_id_for_task(task);
         let safepoint_ttl = self.pause_guard_duration();
         let code = err.error_code().code.to_owned();
-        let msg = err.to_string();
+        let msg = format!("[store = {}] {}", store_id, err);
```

Comment:
nit, maybe `TiKV_store` to distinguish from TiDB node, or do we have a term for TiDB and `store` specifically means TiKV already?